activities/Blockrain.activity/css/activity.css

```diff
@@ -16,6 +16,138 @@
   background-image: url(../icons/view-fullscreen.svg);
 }
 
+#main-toolbar #help-button {
+  background-image: url(../icons/help.svg);
+}
+
+.tutorial-title {
+	background-color: #808080 !important;
```

Comment:
Avoid mixing indentation style in the same file

Reply:
Fixed